### ISSUE TITLE
Is it still working?

### ISSUE THREAD

**Issue by sivka12** [S0Ulle33/telegram-message-cleaner#2]
Запускаю скрипт, ввожу цифру чата, а он трёт вообще все сообщения, включая личные беседы, после чего телеграм выкидывает из всех аккаунтов. Это телеграм что-то поменял, или я что-то делаю не то?

**Comment by fernvenue** [comment]
Can anyone reproduce this?

**Comment by sivka12** [comment]
Yes. But right now I can't even run the script. Immediately after choosing a chat to clear messages, telegram throws me out of all accounts on all devices, and does not allow me to log in for another day.

**Comment by fernvenue** [comment]
> Yes. But right now I can't even run the script. Immediately after choosing a chat to clear messages, telegram throws me out of all accounts on all devices, and does not allow me to log in for another day.

My contact has experienced a similar situation before, but it was not related to running this project. Have you recently registered your Telegram account? If you try to log in using your phone number, does it show "phone number banned"? My contact encountered this situation shortly after registering her account, and the account was deactivated a couple of days later. However, she was able to recover the account by quickly sending an email to Telegram through the help button provided on the login page after the account was canceled.

**Comment by sivka12** [comment]
No, my account is more than five years old. I regularly used this script to clear messages. But since the beginning of this year, the launch of the script began to lead to the exit of account. It seems to me that telegram considers the mass deletion of messages as spam, therefore cancels all account sessions. And after the departure from the telegram accounts, he asks you to enter the code from the letter from the Email, not the SMS.

**Comment by fernvenue** [comment]
I just tried to use this script with a whole new account, it works fine, can not reproduce this issue.

**Comment by sivka12** [comment]
I just tried the script. Again, the departure from all accounts and the requirement to specify the code from the email. It is worth noting that I use the version without tables.

Edit: When trying to log in to the Telegram mobile app , he writes: PHONE_CODE_EXPIRED.
https://github.com/S0Ulle33/telegram-message-cleaner/blob/dev/telegram_message_cleaner.py

**Comment by fernvenue** [comment]
Hi @sivka12, are you disabled Two-Step verification? I think it it required.

**Comment by sivka12** [comment]
Two-factor authentication is enabled, but it wasn't a problem before. I tried disabling it. The script throws it out of the account again after deleting several messages. It is noteworthy that the mobile application writes PHONE_CODE_EXPIRED when logging in. So for normal authentication, I need to reinstall the app.

**Comment by fernvenue** [comment]
That's weird.

**Comment by sivka12** [comment]
I've been writing to telegram support, but they've been ignoring me for months.

**Comment by sivka12** [comment]
Anyway, I checked again. I wrote a python script that deletes one specific message. The message has been deleted. But after that, Telegram immediately logged me out on all devices. Apparently, it's not spam, but the fact of script activation itself.

I don't know what to do. Maybe my account is specifically marked as spam, and now I can't use scripts. Telegram support is not responding.
